terminology.md

```diff
@@ -247,6 +247,59 @@ In Firefox's Web Console the [command line interpreter](https://developer.mozill
 - [REPL](https://en.wikipedia.org/wiki/Read%E2%80%93eval%E2%80%93print_loop)
 .....
 
+### Agent
+
+#### Definition
+An Agent is a thread that holds all state within a JavaScript VM instance.
+
+### Realm
+#### Definition
+A realm consists of a set of [intrinsic](###Intrinsic) objects, an ECMAScript global environment, all of the ECMAScript code that is loaded within the scope of that global environment, and other associated state and resources (i.e. a global object and an associated set of [primordial](###Primoridal) objects). Objects within a realm implicitly share these primordials and can therefore easily disrupt each other by primordial poisoning -- modifying these objects to behave badly. Today, in the browser, realms can be created via same origin iframes.
+#### References
+[ECMA262 Spec](https://tc39.es/ecma262/#sec-code-realms)
+
+
+### Intrinsic
+#### Definition
+All objects that can be observed by a program, whether it is via a global ref, or some other mechanism to access those objects, and then those that can be accessed via syntax/grammar. A reference such as %name% means the intrinsic object.
+#### Example
+`[]`
+#### Reference
+[ECMA262 Spec](https://tc39.es/ecma262/#sec-well-known-intrinsic-objects)
+
+### Primordial
+#### Definition
+Globlal, mutable objects that must exist before any code runs.
+#### Example
+`%ArrayPrototype%`
+
+### Normative
+#### Definition
+The existing way something is done (i.e. status quo)
+
+### SES
+#### Definition
+Secure ECMAScript. A subset of ECMAScript.
+
+### Plenary
+#### Definition
+Meeting of all available TC39 delegates. Occurs around 6 times a year.
+
+
+### Refactoring Hazard
+#### Definition
+A seemingly simple change (i.e. semantically equivalent) that seems like a drop in replacement for an existing piece of code but has specific edge cases that make it not work as expected. As you refactor code from "old pattern" to "new pattern", it would be easy to cause unintentional effects (bugs, etc).
```

Comment:
> (i.e. semantically equivalent)

It wouldn't be a hazard if it was actually equivalent. You might want to say

> (i.e. thought to be semantically equivalent)

instead.

Reply:
Good point. Updated.